### ISSUE TITLE
Cannot drag links between ports with webpack production build

### ISSUE THREAD

**Issue by solussd** [projectstorm/react-diagrams#41]
It works without passing `-p` to webpack. With `-p` I see this in the console: 
```
TypeError: Cannot read property 'diagramEngine' of undefined
    at bundle.js:35
    at bundle.js:29
    at bundle.js:29
    at tr (bundle.js:29)
    at bundle.js:29
    at zr (bundle.js:29)
    at Function.yu [as map] (bundle.js:30)
    at t.render (bundle.js:35)
    at bundle.js:31
    at u (bundle.js:31)
bundle.js:35 Uncaught TypeError: Cannot read property 'model' of null
    at http://localhost:3000/js/bundle.js:35:25977
    at c (http://localhost:3000/js/bundle.js:28:27584)
    at Function.vu (http://localhost:3000/js/bundle.js:30:2172)
    at onMouseUp (http://localhost:3000/js/bundle.js:35:25889)
    at Object.o.invokeGuardedCallback (http://localhost:3000/js/bundle.js:7:17135)
    at a (http://localhost:3000/js/bundle.js:7:10910)
    at Object.u [as executeDispatchesInOrder] (http://localhost:3000/js/bundle.js:7:11161)
    at h (http://localhost:3000/js/bundle.js:6:26323)
    at g (http://localhost:3000/js/bundle.js:6:26449)
    at Array.forEach (native)
```

I know that isn't particularly helpful, but it seems to be an issue caused by symbol name mangling. The diagram does render and I can drag around nodes.

**Comment by dylanvorster** [comment]
will take a look at this later tonight. I have actually turned off name mangling already in the web pack config... so this shouldn't be happening. Are you using the latest?

**Comment by solussd** [comment]
I'm guessing it's mangling, based on the symbol names I'm seeing. I'm not sure that's what it is though.

**Comment by dylanvorster** [comment]
this should be fixed with 2.4.8
